v21.1/import-into.md

```diff
@@ -266,6 +276,50 @@ Google Cloud:
 
 For more detailed information about importing data from Avro and examples, see [Migrate from Avro](migrate-from-avro.html).
 
+### Import into an existing table from a delimited data file
+
+Amazon S3:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> IMPORT INTO customers
+    DELIMITED DATA (
+      's3://your-external-storage/employees.csv?AWS_ACCESS_KEY_ID=[placeholder]&AWS_SECRET_ACCESS_KEY=[placeholder]'
+    )
+    WITH
+      fields_terminated_by='|',
+      fields_enclosed_by='"',
+      fields_escaped_by='"';
```

Comment:
nit: perhaps `\` may be a more typical value for `fields_escaped_by` and may be a better candidate in the example here?